needle/modules/storage/data/keychain_dump.py

```diff
@@ -32,23 +35,54 @@ def module_pre(self):
     # ==================================================================================================================
     def module_run(self):
         # Composing the command string
-        cmd = '{}'.format(self.device.DEVICE_TOOLS['KEYCHAINDUMPER'])
+        cmd = '{}'.format(self.device.DEVICE_TOOLS['KEYCHAIN_DUMP'])

```

Comment:
this creates plist files in the user's home directory. Is there any way to force a specified output directory?